test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsSolutionRestoreServiceTests.cs

```diff
@@ -374,7 +374,7 @@ public async Task NominateProjectAsync_Imports(bool isV2Nomination)
 
             var actualTfi = actualProjectSpec.TargetFrameworks.Single();
             var actualImports = string.Join(";", actualTfi.Imports.Select(x => x.GetShortFolderName()));
-            Assert.Equal("dotnet5.3;portable-net452+win81", actualImports);
+            Assert.Equal("dotnet53;portable-net452+win81", actualImports);
```

Comment:
why did we need to change this?

Reply:
took dotnet tfm out of the list of 4 tfms that we special case to save with periods by default.

Reply:
makes sense. Resolving this.

Reply:
resolving...let me know if you disagree with that change.

Reply:
that was a curiosity question...the codepath is never hit in production :)